### PR TITLE
Fix 987/988

### PR DESCRIPTION
**Pull request by nsrahmad**

### CODE REVIEW

File: Core/clim-core/graph-formatting.lisp [hunk 1]
```diff
@@ -279,7 +279,8 @@
 					   (with-output-to-output-record
 					       (stream 'standard-graph-node-output-record new-node
 						       :object child)
-					     (funcall object-printer child stream))))
+                                             (with-end-of-line-action (stream :allow)
+                                               (funcall object-printer child stream)))))
```
Comment (dkochmanski) [inline Core/clim-core/graph-formatting.lisp]: is it possible for the object to exceed the page too? in that case we shoudl wrap it also in with-end-of-page-action (stram :allow), no?

Comment (nsrahmad) [reply Core/clim-core/graph-formatting.lisp:283]: Yes it is possible, But the default end-of-page-action (:scroll) works fine. I think there is no need to change that.

Comment (dkochmanski) [reply Core/clim-core/graph-formatting.lisp:283]: right you are, thanks

**Comment by dkochmanski** [comment]
merging, thank you

**Pull request merged**